app/build.gradle

```diff
@@ -30,7 +30,6 @@ android {
                 "new ${javaTypes.MAP}()" +
                         "{{" +
                         "put(\"*.staging.5afe.dev\", \"sha256/JSMzqOOrtyOT1kmau6zKhgT676hGgczD5VMdRMyJZFA=,sha256/++MBgDH5WGvL9Bcn5Be30cRcL0f5O+NyoXuWtQdX1aI=\");" +
-                        "put(\"*.gnosis.io\", \"sha256/++MBgDH5WGvL9Bcn5Be30cRcL0f5O+NyoXuWtQdX1aI=\");" +
```

Comment:
Don't we need to add a line for `*.safe.global` here? With the SHA value provided by @safe-global/safe-services ? Otherwise we won't have pinning for these domains, nor?